smdebug/rules/action/action.py

```diff
@@ -0,0 +1,83 @@
+# Standard Library
+import json
+
+# First Party
+from smdebug.core.logger import get_logger
+
+# Local
+from .message_action import MessageAction
+from .stop_training_action import StopTrainingAction
+
+ALLOWED_ACTIONS = ["stoptraining", "sms", "email"]
+
+
+class Actions:
+    def __init__(self, action_str="", rule_name=""):
```

Comment:
I would rename action_str as actions_str just to be clear that the string can contain multiple actions.